03.bowling/bowling.rb

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+FRAMES = 10
+
+def generate_frames
+  pins_text = ARGV[0]
+  pins = pins_text.split(',')
+  frames = []
```

Comment:
すいません、前回のコメントは一部間違っていました。
`each_slice(2).to_a`したときに`[[1,2],[3,4]]`のようなframesができるのであって、9〜17行目の処理ではフレーム分割はしてないですね。
ということは、`frames = []`はframesではなく何なのか？という話になります。

結局、中に入るのはpinだと思うのですが、`pins = []`だと8行目のpinsと名前が被ってしまいます。

10〜17行目の処理は「データをプログラム内で扱いやすくするための処理」だと思うので、normalizeだと言えます。
https://wa3.i-3-i.info/word11632.html

そこで、ここは

```suggestion
  pins = pins_text.split(',')
  normalized_pins = []
```

というように名付けるといいかもしれません。